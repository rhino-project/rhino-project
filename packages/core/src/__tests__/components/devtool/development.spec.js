import { render } from '@testing-library/react';
import { afterEach, describe } from 'vitest';

vi.stubEnv('NODE_ENV', 'development');

describe('RhinoDevTool development', () => {
  afterEach(() => {
    vi.unstubAllEnvs();
  });

  it('should return devtool', async () => {
    const Devtool = await import('../../../components/devtool');
    const { asFragment } = render(<Devtool.RhinoDevTool />);
    expect(asFragment()).toMatchSnapshot();
  });
});
