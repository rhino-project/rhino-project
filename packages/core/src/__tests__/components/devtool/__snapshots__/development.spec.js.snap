// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`RhinoDevTool development > should return devtool 1`] = `
<DocumentFragment>
  <button
    class="_toggleButton_bb2097"
  >
    <img
      alt="Toggle"
      height="40"
      src="/src/components/devtool/rhinoRedLogo.png"
      width="40"
    />
  </button>
  <div
    class="_rhinoTool_bb2097 _collapsed_bb2097"
  />
</DocumentFragment>
`;
